### PR TITLE
[x-1]-thread local

### PR DESCRIPTION
**Pull request by Jokeroff**

### CODE REVIEW

File: src/main/java/com/lebediev/movieland/web/controller/interceptor/LoggingInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import javax.servlet.http.HttpServletResponse;
+import java.util.UUID;
+
+import static com.lebediev.movieland.service.authentication.AuthService.getUserThreadLocal;
+
+public class LoggingInterceptor extends HandlerInterceptorAdapter {
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+        User user = getUserThreadLocal();
+        String userName = "guest";
```
Comment (TakeALookHere) [inline src/main/java/com/lebediev/movieland/web/controller/interceptor/LoggingInterceptor.java]: should be constant

File: src/test/java/com/lebediev/movieland/dao/jdbc/integrationtests/JdbcMovieDaoITest.java [hunk 2]
```diff
@@ -85,5 +86,54 @@ public void testGetMovieById() {
...
+    }
+
+    @Test
+    public void testUpdate() {
+        MovieDtoForUpdate actual = new MovieDtoForUpdate();
+        actual.setNameNative("before update");
+        actual.setYearOfRelease(2000);
+        actual.setCountries(new int[]{1, 2, 3});
+        actual.setGenres(new int[]{4, 5});
+        jdbcMovieDao.add(actual);
```
Comment (TakeALookHere) [inline src/test/java/com/lebediev/movieland/dao/jdbc/integrationtests/JdbcMovieDaoITest.java:116]: i think that this method returns id of added movie, so below you don't need after update method select by nativeName: use returned id for further checks

File: src/main/java/com/lebediev/movieland/web/controller/dto/MovieDtoForUpdate.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
+package com.lebediev.movieland.web.controller.dto;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MovieDtoForUpdate {
```
Comment (TakeALookHere) [inline src/main/java/com/lebediev/movieland/web/controller/dto/MovieDtoForUpdate.java:6]: i think that we have the same class MovieDto with the same kind of fields. If you think about int[ ], adding a constructor without parameters will solve this.

Comment (Jokeroff) [reply src/main/java/com/lebediev/movieland/web/controller/dto/MovieDtoForUpdate.java:6]: MovieDto already has "not simple" structure with JsonViews for 3 different scenarios (

File: src/test/java/com/lebediev/movieland/dao/jdbc/integrationtests/JdbcMovieDaoITest.java (hunk 2 above)
Comment (Jokeroff) [reply src/test/java/com/lebediev/movieland/dao/jdbc/integrationtests/JdbcMovieDaoITest.java:116]: it returns id inside, not visible outside

File: src/main/java/com/lebediev/movieland/web/controller/interceptor/LoggingInterceptor.java (hunk 1 above)
Comment (Jokeroff) [reply src/main/java/com/lebediev/movieland/web/controller/interceptor/LoggingInterceptor.java]: it is reused

**Pull request merged**